src/searches/issueAttachments.ts

```diff
@@ -28,72 +27,73 @@ interface issueAttachmentsRequestResponse {
           }
           attachments {
             nodes {
+              id,
+              title,
+              subtitle,
               metadata
             }
           }
         }
       }`;
 
-      z.console.log(JSON.stringify(query));
```

Comment:
NIT: the diff is not super easy to read as all indentation has changed

Reply:
Prettier ran automatically, but you can hide whitespaces in the review:
<img width="289" alt="Screenshot 2022-03-23 at 13 22 13" src="https://user-images.githubusercontent.com/674084/159698117-83783a0c-660e-415f-9ad2-7b6553d02a80.png">

Reply:
For the sake of consistency, I've reset formatting to what we have on `master`